test/integration/user_commands.cc

```diff
@@ -693,8 +693,8 @@ TEST_F(UserCommandsTest, Light)
 {
   // Start server
   ServerConfig serverConfig;
-  const auto sdfFile = ignition::common::joinPaths(
-    std::string(PROJECT_SOURCE_PATH), "test", "worlds", "lights_render.sdf");
+  const auto sdfFile = std::string(PROJECT_SOURCE_PATH) +
+    "/test/worlds/lights_render.sdf";
```

Comment:
Using `joinPaths` is better because it supports Windows too. Was that not working?